From cb56859f7e86f3f9d83a5666b9430d41055dd871 Mon Sep 17 00:00:00 2001
From: Peng Fan <peng.fan@nxp.com>
Date: Mon, 9 Aug 2021 19:08:15 +0800
Subject: [PATCH 17/19] LF-4276 arm: dts: imx8mq: add the dwc3 fallback
 compatible

Add the dwc3 fallback compatible to make distro could
recognize usb controller

Reviewed-by: Ye Li <ye.li@nxp.com>
Signed-off-by: Peng Fan <peng.fan@nxp.com>
---
 arch/arm/dts/imx8mq.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/imx8mq.dtsi b/arch/arm/dts/imx8mq.dtsi
index 0ed72c382c..2c206c8866 100644
--- a/arch/arm/dts/imx8mq.dtsi
+++ b/arch/arm/dts/imx8mq.dtsi
@@ -1230,7 +1230,7 @@
 		};
 
 		usb_dwc3_0: usb@38100000 {
-			compatible = "fsl,imx8mq-dwc3";
+			compatible = "fsl,imx8mq-dwc3", "snps,dwc3";
 			reg = <0x38100000 0x10000>;
 			clocks = <&clk IMX8MQ_CLK_USB1_CTRL_ROOT>,
 			         <&clk IMX8MQ_CLK_USB_CORE_REF>,
@@ -1263,7 +1263,7 @@
 		};
 
 		usb_dwc3_1: usb@38200000 {
-			compatible = "fsl,imx8mq-dwc3";
+			compatible = "fsl,imx8mq-dwc3", "snps,dwc3";
 			reg = <0x38200000 0x10000>;
 			clocks = <&clk IMX8MQ_CLK_USB2_CTRL_ROOT>,
 			         <&clk IMX8MQ_CLK_USB_CORE_REF>,
-- 
2.33.0

